apps/showcase/src/components/showcase/component-replacement/component-replacement-pres.template.html

```diff
@@ -0,0 +1,9 @@
+<div class="card p-2">
+  <div class="w-50">
+    <p>Gregorian date: <b>{{(dateFormControl.valueChanges | async) || initialDate}}</b></p>
+  </div>
+  <div class="w-50">
+    <ng-template c11n [component]="presenter$ | async" [inputs]="{id: 'date-outbound'}" [formControl]="dateFormControl"
+      [config]="{}" [outputs]="outputs"></ng-template>
```

Comment:
Do you have to specify `config` and `outputs` if we don't use them in this example ?

Reply:
good point. They are not mandatory so I removed them.